reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerRequest.java

```diff
@@ -143,6 +144,13 @@ default Flux<HttpContent> receiveContent() {
 	 */
 	HttpHeaders requestHeaders();
 
+	/**
+	 * return parsed and decoded query parameter name value pairs
+	 *
+	 * @return query parameters {@link Map&lt;String,List&lt;String&gt;&gt;}
+	 */
```

Comment:
```suggestion
	 * @since 1.1.4
	 */
```

can you please add the @since annotation ?
Take care: I have set the milestone for this PR to 1.1.4. But, when Violeta will do her review, maybe she will want to backport this to the 1.0.x; in this case we'll have to use `@since 1.0.29`